packages/angular/cli/lib/config/schema.json

```diff
@@ -605,6 +605,11 @@
               "description": "Output sourcemaps.",
```

Comment:
Apparently, this is also required apart from the schema of the specific architect buiilder.